From: SeongJae Park <sj@kernel.org>
Date: Wed, 25 Dec 2024 12:41:05 -0800
Subject: [PATCH] ==== DAMON docs update for 6.14 ====

Docs/mm/damon: add tuning guide and misc updates

Add DAMON monitoring parameters tuning guide, and misc changes for
fixups and clarifications of the documents.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 damon_meta_changes/TyBQIHnJ | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 damon_meta_changes/TyBQIHnJ

diff --git a/damon_meta_changes/TyBQIHnJ b/damon_meta_changes/TyBQIHnJ
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

